napi-inl.h

```diff
@@ -3823,7 +3839,16 @@ inline napi_value ObjectWrap<T>::InstanceSetterCallbackWrapper(
 }
 
 template <typename T>
-inline void ObjectWrap<T>::FinalizeCallback(napi_env env, void* data, void* /*hint*/) {
+inline void ObjectWrap<T>::FinalizeCallback(napi_env env, void* data, void* hint) {
```

Comment:
IINM, if `napi_wrap()` fails then this will never be called, and `zombie` will never be freed, resulting in a leak.